<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KNewPasswordDialog</class>
 <widget class="QDialog" name="KNewPasswordDialog">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="KIconWidget" name="iconWidget" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="labelPrompt">
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KMessageWidget" name="statusMsgWidget"/>
   </item>
   <item>
    <widget class="KNewPasswordWidget" name="pwdWidget" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KNewPasswordWidget</class>
   <extends>QWidget</extends>
   <header>knewpasswordwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KIconWidget</class>
   <extends>QWidget</extends>
   <header>kiconwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KNewPasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KNewPasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
